tests/integration/test_clip.py

```diff
@@ -140,6 +140,18 @@ def test_out_ndim():
     assert np.array_equal(out_np, out_num)
 
 
+def test_out_np_array():
+    array = ((2, 3, 4), (3, 4, 5), (6, 6, 12))
+    amin = (2, 3, 1)
+    amax = 6
+    np_arr = np.array(array)
+    num_arr = num.array(array)
+    out_np = out_num = np.zeros(np_arr.shape)
```

Comment:
This seems like a mistake. This only makes a single "out" array, and points both variable names at that same array:
```
In [1]: import numpy as np

In [2]: out_np = out_num = np.zeros(10)

In [3]: out_np is out_num
Out[3]: True
```
That means the comparison at the end is a self-comparison